tests/integration/apps/ruby/Gemfile.lock

```diff
@@ -54,14 +54,14 @@ GEM
     ruby2_keywords (0.0.5)
     sinatra (3.0.5)
       mustermann (~> 3.0)
-      rack (~> 2.2, >= 2.2.4)
+      rack (~> 3.0.1)
```

Comment:
I worked on a security story recently about upgrading Rack and if I recall Rack 3 was not compatible with Sinatra, which was causing CI errors for the Ruby buildpack. Looks like even the [latest version of Sinatra](https://rubygems.org/gems/sinatra/versions/3.0.6) is expecting Rack 2.2.x

I suspect we would have to migrate the Ruby test app to use some other Ruby web app gem to get it working on Rack 3